src/lib/components/FormModel/children/Viewer/UI/ui-mapping.js

```diff
@@ -1,13 +1,11 @@
 import React from 'react';
 import { UITypes } from '@constants';
 
-import SingleLine from './SingleLine';
+import {SingleLine} from './SingleLine';
 import MultiLine from './MultiLine';
 import DateLine from './DateLine';
-import BoolLine from './BoolLine';
+import { BoolLine } from './BoolLine';
 import ColorLine from './ColorLine';
-import Heading from '../../Form/UI/Heading';
```

Comment:
Are Heading/Comment no longer in the Viewer, or is it generated another way?

Reply:
I think when we spoke yesterday we determined this file isn't actually used anymore?